3d/generate_2d.py

```diff
@@ -45,25 +46,29 @@
     svg_output = renderer.render_svgs()
 
     # Export to png
-    raster_svg = os.path.join(laser_parts_directory, 'raster.svg')
-    raster_png = os.path.join(laser_parts_directory, 'raster.png')
+    cwd = os.getcwd()
```

Comment:
Is specifying cwd necessary? I would have thought that in absence of an absolute path, any path would be treated as relative to cwd already?

I think there may be some existing issues with running this script from outside of the `3d` directory, but to resolve that kind of issue I think we'd want to make the paths relative to this file's location, e.g. make `laser_parts_directory` relative to this file, something like:
`laser_parts_directory = os.path.join(os.path.dirname(__file__), 'build', 'laser_parts')`.

That way even if you call the script from `/Users/foobar/` it will still output to the `3d/build/` folder within the repo (which is gitignored). Doing that would probably also require updating the `input_file` given to the `Renderer` above to properly reference the 'splitflap.scad' location as well.

Anyway, I think updating this script to work from outside the `3d` directory is worthwhile to do at some point (other scripts like `generate_stl.py` already support that), but probably a larger change, so for the purposes of this PR I'm wondering if this cwd specification is actually necessary, or if just the `commands` change when invoking inkscape/openscad is sufficient to make this work on OS X?